src/dotnet/commands/dotnet-vstest/VSTestForwardingApp.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.Cli
+{
+    public class VSTestForwardingApp
```

Comment:
I think the composition is good. Makes it easier to reason about the behavior here without looking into ForwardingApp